npm link sometimes needs root

My system node modules directory is only writable by root, so when I run `create-ink-app` as a regular user it fails with:

```
$ npx create-ink-app
npx: installed 260 in 8.762s

  ✔ Copy files
  ✔ Install dependencies
  ✖ Link executable
    → npm ERR!     /home/mgalgs/.npm/_logs/2019-10-08T00_44_52_001Z-debug.log
Error: Command failed: npm link
npm ERR! path /home/mgalgs/src/kbrowse
npm ERR! code EACCES
npm ERR! errno -13
npm ERR! syscall symlink
npm ERR! Error: EACCES: permission denied, symlink '/home/mgalgs/src/kbrowse' -> '/usr/lib/node_modules/kbrowse'
npm ERR!  { [Error: EACCES: permission denied, symlink '/home/mgalgs/src/kbrowse' -> '/usr/lib/node_modules/kbrowse']
npm ERR!   stack:
npm ERR!    'Error: EACCES: permission denied, symlink \'/home/mgalgs/src/kbrowse\' -> \'/usr/lib/node_modules/kbrowse\'',
npm ERR!   errno: -13,
npm ERR!   code: 'EACCES',
npm ERR!   syscall: 'symlink',
npm ERR!   path: '/home/mgalgs/src/kbrowse',
npm ERR!   dest: '/usr/lib/node_modules/kbrowse' }
npm ERR!
npm ERR! The operation was rejected by your operating system.
npm ERR! It is likely you do not have the permissions to access this file as the current user
npm ERR!
npm ERR! If you believe this might be a permissions issue, please double-check the
npm ERR! permissions of the file and its containing directories, or try running
npm ERR! the command again as root/Administrator (though this is not recommended).

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/mgalgs/.npm/_logs/2019-10-08T00_44_52_001Z-debug.log


    at makeError (/home/mgalgs/.npm/_npx/17549/lib/node_modules/create-ink-app/node_modules/execa/index.js:174:9)
    at Promise.all.then.arr (/home/mgalgs/.npm/_npx/17549/lib/node_modules/create-ink-app/node_modules/execa/index.js:278:16)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

Is it really necessary to `link` the package on initial creation?  Or should that be a separate step (left for the user to execute if/when it's appropriate)?

--- a/index.js
+++ b/index.js
@@ -130,12 +130,17 @@ module.exports = () => {
 		},
 		{
 			title: 'Link executable',
-			task: async () => {
+			task: async (_, task) => {
 				if (useTypeScript) {
 					await execa('npm', ['run', 'build']);
 				}
 
-				return execa('npm', ['link']);
+				try {
+					await execa('npm', ['link']);
+					// eslint-disable-next-line unicorn/prefer-optional-catch-binding
+				} catch (_) {
+					task.skip('npm link failed, please try running with sudo');
+				}
 			}
 		}
 	]);

